solana/programs/example-native-token-transfers/tests/governance.rs

```diff
@@ -131,6 +131,50 @@ async fn test_governance() {
     assert!(config_account.paused);
 }
 
+#[tokio::test]
+async fn test_governance_one_step_transfer() {
+    let (mut ctx, test_data) = setup(Mode::Locking).await;
+
+    let governance_pda = test_data.governance.governance();
+
+    // step 1. transfer ownership to governance (1 step)
+    let ix = example_native_token_transfers::instruction::TransferOwnershipOneStep;
+
+    let accs = example_native_token_transfers::accounts::TransferOwnership {
+        config: test_data.ntt.config(),
+        owner: test_data.program_owner.pubkey(),
+        new_owner: governance_pda,
+        upgrade_lock: test_data.ntt.upgrade_lock(),
+        program_data: test_data.ntt.program_data(),
+        bpf_loader_upgradeable_program: bpf_loader_upgradeable::id(),
+    };
+
+    Instruction {
+        program_id: test_data.ntt.program,
+        accounts: accs.to_account_metas(None),
+        data: ix.data(),
+    }
+    .submit_with_signers(&[&test_data.program_owner], &mut ctx)
+    .await
+    .unwrap();
+
+    // step 2. set paused
+    wrap_governance(
+        &mut ctx,
+        &test_data.governance,
+        &test_data.ntt.wormhole,
+        set_paused(&test_data.ntt, SetPaused { owner: OWNER }, true),
```

Comment:
Should `OWNER` be `governance_pda` here? The test works on my machine so I guess not, but I'm a bit confused reading the code. (I'll dig around test suite more to figure out what's going on but this line looks a little odd at first glance given that we've just changed the owner)

Reply:
Ah OK so OWNER isn't constant it's like a placeholder for whatever the actual owner value is at a given time? Interesting

Reply:
yeah, it's replaced here
https://github.com/wormhole-foundation/example-native-token-transfers/blob/4f165a3d8c743e543fbc38e958185aad06f22d91/solana/programs/wormhole-governance/src/instructions/governance.rs#L464-L470